game/gothic.cpp

```diff
@@ -100,13 +103,22 @@ Gothic::Gothic() {
     modFile.reset(new IniFile(mod));
     }
 
+  std::vector<std::u16string> modvdfs;
   if(modFile!=nullptr) {
     wrldDef = modFile->getS("SETTINGS","WORLD");
     size_t split = wrldDef.rfind('\\');
     if(split!=std::string::npos)
       wrldDef = wrldDef.substr(split+1);
     plDef = modFile->getS("SETTINGS","PLAYER");
+
+    std::u16string vdf = TextCodec::toUtf16(std::string(modFile->getS("FILES","VDF")));
```

Comment:
As already mentioned in comments:
would be nice avoid `basic_stringstream` completely. I've tried to read cppreference, still can't say what is mandatory and what part of streams are optional in library :D 

Since you have string value - you can simply "for-loop" it.